src/test/js/multibranch/noRunsActivityPage.js

```diff
@@ -0,0 +1,24 @@
+const jobName = 'noRunsActivityPage';
+/** @module failing
+ * @memberof testcases
+ * @description TEST: basic tests around the failing pipeline. Test whether the result is not collapsed.
+ */
+module.exports = {
+    /**
+     * Create Multibranch Pipeline Job "noRunsActivityPage" with no branches.
+     * @param browser
+     */
+    'Step 01': function (browser) {
+        var multibranchCreate = browser.page.multibranchCreate().navigate();
+      
+        multibranchCreate.createBranch(jobName);
+    },
+    /**
+     * Make sure that we show the empty state.
+     */
+    'Step 02': function (browser) {
+         var blueActivityPage = browser.page.bluePipelineActivity().forJob(jobName, 'jenkins');
+
+         blueActivityPage.waitForElementVisible('@emptyStateShoes');
+    }
+}
```

Comment:
@imeredith In fact ... there's already an [bluePipelineActivity#assertEmptyLayoutOkay](https://github.com/jenkinsci/blueocean-acceptance-test/blob/master/src/main/js/page_objects/blueocean/bluePipelineActivity.js#L59-L66). That you could have called. It doesn't do a lot more than check for the element, but it probably could and if it was enhanced over time then your test would benefit from that